dll/cpl/timedate/internettime.c

```diff
@@ -338,18 +273,20 @@ InetTimePageProc(HWND hwndDlg,
             {
                 case IDC_UPDATEBUTTON:
                 {
-                    ULONG ulTime;
+                    DWORD dwError;
 
                     SetNTPServer(hwndDlg);
 
-                    ulTime = GetTimeFromServer();
-                    if (ulTime != 0)
-                        UpdateSystemTime(ulTime);
+                    dwError = W32TimeSyncNow(L"localhost", 0, 0);
```

Comment:
Most of the reference that I have seen use "NULL" instead of "localhost", but this seems to be just as good.

Reply:
Hmm I seem to recall it didn't work for me with NULL. I'll try it -- if NULL works, I'd prefer that.

Reply:
See [CORE-13001](https://jira.reactos.org/browse/CORE-13001) for ClockResync test program that uses NULL.